### PR TITLE
Adds helper utility for generating plots from the neuron paper

### LINKED ISSUE #604: Make sequence learning script easily replicate results
This issue is to allow a new user to easily replicate the plots in the Neuron paper.  Two things would be helpful here:

- Update the directory to include scripts that generate nice looking plots

- Clearly document the exact commands to https://github.com/numenta/nupic.research/tree/master/projects/sequence_learning to replicate all the plots in the Neuron paper.

### PR DESCRIPTION
**Pull request by oxtopus**
Fixes #604

@subutai Here are the changes for the plots only using the existing phases implementation.  I can replicate the original results from the paper with plots that are consistent.

For example:

figure_6b

### CODE REVIEW

**Comment by oxtopus** [comment]
@subutai I'd like to merge this so I can focus on swapping out the implementation separately and not conflate the issues.  With these changes, you can generate pretty plots suitable for publishing, and are helpful in debugging the implementation.

File: projects/sequence_learning/README.md [hunk 1]
```diff
@@ -34,4 +34,22 @@ the network to damage. After the network reached stable performance we
...
+Generating plots
+----------------
+
+A helper utility `generate_plots.py` is provided to assist in reproducing
+figure 6A-B from the neuron paper.
+
+### Figure 6A
+
+```
+python generate_plots.py 0.0 --figure A --passthru="--name Fig6A --simulation killer"
```
Comment (subutai) [inline projects/sequence_learning/README.md]: Should not be simulation "killer" for Fig6A

File: projects/sequence_learning/sequence_simulations.py [hunk 2]
```diff
@@ -132,6 +132,14 @@ def computePredictionAccuracy(pac, pic):
     return (pac / pcols)
 
 
+def plotAccuracies(accuracies):
```
Comment (subutai) [inline projects/sequence_learning/sequence_simulations.py]: This function seems redundant now - can we remove it?

File: projects/sequence_learning/generate_plots.py [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+
+parser = OptionParser("python %prog noise [noise ...]")
+parser.add_option("--figure",
+                  help="Which figure to plot.  Must be 'A' or 'B'.")
+parser.add_option("--passthru",
+                  help=("Pass options through to sequence_simulations.py.  "
+                        "See `python sequence_simulations.py --help` for "
+                        "options"))
+
```
Comment (subutai) [inline projects/sequence_learning/generate_plots.py]: Should these lines be within the "__main__" block below?

**Comment by subutai** [comment]
@oxtopus Agreed - it would be good to merge this. Looks good overall - left a few minor comments.

**Comment by oxtopus** [comment]
Thanks, @subutai.  I removed the redundant plot functionality and put the parser operations back in the main block.

**Pull request merged**